src/modules/kafka/test-certs/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM openjdk:8-slim
```

Comment:
The `test-certs` directory will need to be excluded in the `tsconfig.json`, to avoid it being packaged

Reply:
Good point. Done.